src/ray/raylet/worker_pool.cc

```diff
@@ -1055,13 +1055,13 @@ void WorkerPool::TryKillingIdleWorkers() {
     }
 
     const auto &job_id = idle_worker->GetAssignedJobId();
-    if (finished_jobs_.count(job_id) > 0) {
+    if (finished_jobs_.contains(job_id)) {
       // The job has finished, so we should kill the worker immediately.
       KillIdleWorker(idle_worker, it->second);
       it = idle_of_all_languages_.erase(it);
     } else {
-      if (it->second == -1 ||
-          now - it->second >
+      if (it->second == absl::Time{} ||
```

Comment:
can we use absl::Time calculations:

now - it->second > absl::Milliseconds(config::my_ms())

Reply:
OK, but my implementation is doing the same thing? no strong opinion though.